README.adoc

```diff
@@ -1527,6 +1527,34 @@ else
 end
 ----
 
+=== `unless` with Multiple Conditions
+
+Do not use `unless` with multiple conditions.
+
+[source,ruby]
+----
+# bad
+unless sweet?(fruit) && !fruit.green?
+  yield fruit # skip yielding sweet non-green fruits
+end
```

Comment:
This is taken from the same source code as an example below 😉, just changed the names :spy: